provider/redtiger/test/integration_test.go

```diff
@@ -146,52 +146,48 @@ func (s *RedTigerIntegrationTestSuite) Test_MAKES_A_BET_REQUEST_WITH_STAKE_0() {
 
 func (s *RedTigerIntegrationTestSuite) Test_MAKES_SEVERAL_BET_REQUESTS_WITH_SAME_ROUNDID() {
 	roundID := rnd()
-	_, err := s.client.Stake(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0))
+	_, err := s.client.Stake(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0), nil)
 	s.Require().NoError(err, "Request should not produce hard error")
 
 	// Firing twice with the same round ID should be allowed as long as the round is not ended
-	resp, err := s.client.Stake(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0))
+	resp, err := s.client.Stake(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0), nil)
 	s.Require().NoError(err, "Request should not produce hard error")
 	s.Assert().True(resp.Success)
 }
 
 func (s *RedTigerIntegrationTestSuite) Test_MAKES_SEVERAL_BET_REQUESTS_WITH_SAME_ROUNDID_AFTER_PAYOUT() {
 	roundID := rnd()
-	_, err := s.client.Stake(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0))
+	_, err := s.client.Stake(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0), nil)
 	s.Require().NoError(err, "Request should not produce hard error")
 
 	_, err = s.client.Payout(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0), toMoney(0.0), toJackpotMoney(0.0))
 	s.Require().NoError(err, "Request should not produce hard error")
 
 	// Firing twice with the same round ID should not be allowed if the round is is ended
-	resp, err := s.client.Stake(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0))
+	resp, err := s.client.Stake(gameID, rnd(), roundID, toMoney(10.0), toMoney(0.0), nil)
 	s.Require().Error(err, "Request should produce hard error")
 	s.Assert().False(resp.Success)
 }
 
 func (s *RedTigerIntegrationTestSuite) Test_MAKES_A_BET_REQUEST_WITH_RECON_TOKEN() {
 	s.client.baseRequest.Token = s.ProviderConfig.Auth["recon_token"].(string)
-	resp, err := s.client.Stake(gameID, rnd(), rnd(), toMoney(10.0), toMoney(0.0))
+	resp, err := s.client.Stake(gameID, rnd(), rnd(), toMoney(10.0), toMoney(0.0), nil)
 	s.Require().Error(err, "Request should produce hard error")
 	s.Assert().False(resp.Success)
 	s.Assert().Equal(redtiger.NotAuthorized, resp.Error.Code)
 }
 
 func (s *RedTigerIntegrationTestSuite) Test_MAKES_A_BET_REQUEST_WITH_PROMO_PARAMS() {
-	// TODO is this correct? SHould it be PromoBuyin or simply a stake but add the Promo fields?
```

Comment:
I agree with this comment. Besides, PromoBuyin is tested in another test in this file